input/fsh/OrganizationSECommonsLite.fsh

```diff
@@ -84,4 +84,76 @@ Description: """This is a light weight Swedish Organization profile.
 
 * address.line.extension contains
             iso21090-ADXP-careOf named careOf 0..* MS
-        and iso21090-ADXP-streetAddressLine named streetAddressLine 0..* MS
\ No newline at end of file
+        and iso21090-ADXP-streetAddressLine named streetAddressLine 0..* MS
+
+Instance: minimal-organization
+InstanceOf: OrganizationSECommonsLite
+Description: "Minimal Organization instance"
+Usage: #example
+* meta.profile = Canonical(OrganizationSECommonsLite)
+* identifier
+  * use = #official
+  * type = IDENTIFIER_TYPE#XX
+  * system = "urn:oid:2.5.4.97"
+  * value = "5566778899"
+* name = "Svenska Kliniken AB"
+* type[0].coding = SCT#43741000 "Vårdenhet"
+  * version = "http://snomed.info/sct/45991000052106/version/20220531"
+* type[+].coding = SCT#143591000052106 "Vårdgivare"
+  * version = "http://snomed.info/sct/45991000052106/version/20220531"
+* telecom[0]
+  * system = #phone
+  * value = "+46101234567"
+* telecom[+]
+  * system = #email
+  * value = "info@svenskakliniken.se"
+* address
+  * line = "Storgatan 1"
+  * city = "Stockholm"
+  * state = "Stockholms län"
+  * postalCode = "100 55"
+  * country = "SE"
+
+Instance: maximal-organization
+InstanceOf: OrganizationSECommonsLite
+Description: "Maximal Organization instance"
+Usage: #example
+* meta.profile = Canonical(OrganizationSECommonsLite)
+* identifier[0]
+  * use = #official
+  * type = IDENTIFIER_TYPE#PRN
+  * system = "urn:oid:1.2.752.29.4.19"
+  * value = "HSA-ID-EXAMPLE-1234"
+* identifier[+]
+  * use = #official
+  * type = IDENTIFIER_TYPE#XX
```

Comment:
According to the profile this should be:

http://terminology.hl7.org/CodeSystem/v2-0203#XX

Reply:
Which I know see is solved using aliases :)